webapp/sources/rudder/rudder-web/src/main/elm/onboarding/sources/DataTypes.elm

```diff
@@ -0,0 +1,44 @@
+module DataTypes exposing (..)
+
+import Http exposing (Error)
+import List exposing (..)
+
+type Section
+  = Welcome SectionState
+  | Account SectionState AccountSettings
+  | Metrics SectionState String
```

Comment:
You should have a MetricsState type which has 3 or 4 (depending if you want to include a NotDefined) values : 

* NoMetrics
* Complete
* Minimal
* (Not defined) => or three states but you need a Maybe MetricsState